.github/workflows/tests-main.yml

```
@@ -37,7 +37,8 @@ jobs:
     if: |
       needs.check-api-folder-changes.outputs.folder_changed == 'true' ||
       needs.check-pro-folder-changes.outputs.folder_changed == 'true' ||
-      github.ref == 'refs/heads/master' 
+      github.ref == 'refs/heads/master'  ||
```

Il y a 2 espaces au lieu d'un avant le `||`. J'ai failli tomber de ma chaise ! :) 

```suggestion
      github.ref == 'refs/heads/master' ||
```

En vrai, j'approuve quand même : je survivrai à cette abomination typographique. ;)

(idem plus bas)

---

c'est corrigé, merci pour la remarque